plugins/traction_innkeeper/traction_innkeeper/v1_0/innkeeper/tenant_manager.py

```diff
@@ -168,6 +172,8 @@ async def create_tenant(
                     auto_issuer=auto_issuer,
                 )
                 await tenant.save(session, reason="New tenant")
+                new_tenants = await tenant.query(session)
+                self._logger.error(f"The current list of tenants is {new_tenants}")
```

Comment:
Not sure if this is meant to be left in, or at level `error`?

Reply:
Meant to drop this logging. Thanks for catching that